app/views/content/non-uk-teachers/visas-for-non-UK-trainees.md

```diff
@@ -0,0 +1,257 @@
+
+Use [find postgraduate teacher training](https://www.gov.uk/student-visa/apply) to find courses offered by providers where you can be sponsored for a skilled worker visas. Currently, places on these courses are very limited. Before you apply for a salaried teacher training place, we suggest you contact the training provider to check you can be sponsored.
+
+You’ll be able to apply for your visa if:
+
+* you have a confirmed offer for a salaried training place 
+* your training provider has confirmed you'll be employed by a school which is a licensed skilled worker sponsor
+* [you can speak, read, write, and understand English](https://www.gov.uk/skilled-worker-visa/knowledge-of-english)
+* your salary meets the requirement for a skilled worker visa
+
+Your training provider can help you submit the correct details for your visa application. 
+
+For your teacher training place offer to be confirmed, you’ll have to pass criminal record and health checks, so we recommend applying to your training provider early.
+
+#### Salary requirement for a skilled worker visa 
+
+The salary threshold you must meet depends on the region you work in as a salaried trainee.
+
+#### Full-time training
+
+Non-UK salaried trainee teachers must earn a minimum salary to be eligible for a skilled worker visa – this varies by region, as follows:
+
+* inner London: £23,099
+* outer London: £21,832
+* London fringe: £19,613
+* rest of England: £18,419
+
+Check the region you will be working in with your teacher training provider.
+
+#### Part-time training
+
+You may be able to train part-time, but your salary must meet the threshold for a skilled worker visa.
+
+There are 2 requirement:
+
+Your part-time salary must be at least £20,480.
+
+The full-time equivalent of your salary must meet the threshold set for the skilled worker visa in your region, as follows: 
+
+* inner London: £23,099
+* outer London: £21,832
+* London fringe: £19,613
+* rest of England: £18,419
+
+Check the region you will be working in with your teacher training provider.
+
+<p class="call-to-action__action">
+  <a href="https://www.gov.uk/skilled-worker-visa">Apply for a skilled worker visa<span></span></a>
+</p>
+
+### Postgraduate teaching apprenticeships
+
+To apply for a postgraduate teaching apprenticeship you will need:
+
+* settled or pre-settled status under the EU Settlement Scheme
+* to have been living in the UK or the EEA for 3 or more years before the start of your apprenticeship
+
+If you do not have settled or pre-settled status under the EU Settlement Scheme, you will need:
+
+* a visa or immigration status allowing you to work in the UK
+* to have been living in the UK for 3 or more years before the start of your apprenticeship
+
+## Other types of visa you can use to work and study in England
+
+### Graduate visa
+
+If you are already an international student in the UK, you can [apply for a graduate visa on successful completion of your degree](https://www.gov.uk/graduate-visa).
```

Comment:
I would either move the link to the end of the section or reduce the content to a single signposting line.

Reply:
OK :)